release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -3638,3 +3638,25 @@ $ oc adm release info 4.8.51 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-8-52"]
+=== RHBA-2022:7034 - {product-title} 4.8.52 bug fix update
+
+Issued: 2022-10-26
+
+{product-title} release 4.8.52 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:7034[RHBA-2022:7034] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:7032[RHBA-2022:7032] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.8.52 --pullspecs
+----
+
+[id="ocp-4-8-52-bug-fixes"]
+==== Bug fixes
+* Previously, the liveness probe woud timeout after one second, causing the kuryr-controller to restart. This fix increases the default time limit, resulting in kuryr-controller running for a longer period. (link:https://issues.redhat.com/browse/OCPBUGS-472?jql=key%20in%20(%27OCPBUGS-472%27%2C%27OCPBUGS-1977%27%2C%27OCPBUGS-1023%27)[*OCPBUGS-216*])
```

Comment:
I'd add the missing lowercase-L to "woud"

Reply:
Thanks for catching, James! I have made the updates! :)